Doc/library/csv.rst

```diff
@@ -233,7 +233,12 @@ The :mod:`csv` module defines the following classes:
    information for how to handle doublequotes, whitespace, delimiters, etc.
    Due to the lack of a strict CSV specification, different applications
    produce subtly different CSV data.  :class:`Dialect` instances define how
-   :class:`reader` and :class:`writer` instances behave.
+   :class:`reader` and :class:`writer` instances behave. Note that the default
```

Comment:
The above can use rewriting and some of the new text belongs in the Dialects and Formatting Parameters section.  I will try to submit an update.